maps/VermilionPort.asm

```diff
@@ -84,12 +84,6 @@ VermilionPortWalkUpToShipScript:
 	iftrue .skip
 	turnobject PLAYER, LEFT
 	opentext
-	readvar VAR_WEEKDAY
-	ifequal MONDAY, .NextShipWednesday
```

Comment:
For now I suggest keeping these in case its decided that this change should be locked behind a flag. I suspect that it probably won't need to be though, so once this gets properly merged into the main branch for speedchoice V7 we should propose to delete this.